src/components/TeamSection/TeamSection.styles.ts

```diff
@@ -1,5 +1,19 @@
 import styled from "styled-components/macro";
 
+// helper function 
```

Comment:
I think we've used RGB and hex codes in our codebase plenty of times before, but I think we should move away from directly having these in our code. Would it be possible to store the RGB in a variable and use the variable instead?

Reply:
You are totally right, I had copied them from browser console and forgot to change them into var. Addressed!